x-pack/legacy/plugins/snapshot_restore/public/app/components/repository_delete_provider.tsx

```diff
@@ -49,7 +49,9 @@ export const RepositoryDeleteProvider: React.FunctionComponent<Props> = ({ child
 
   const deleteRepository = () => {
     const repositoriesToDelete = [...repositoryNames];
-    deleteRepositories(repositoriesToDelete).then(({ data: { itemsDeleted, errors }, error }) => {
```

Comment:
while working on policy deletion, I found that for server errors such as `404` or `500`, which don't return a `data` object, early destructuring here causes an undefined object error, so no toast notification is shown. I adjusted this for policy deletion and fixed it for other delete providers in this app such as here.